install.md

```diff
@@ -190,11 +190,14 @@ To build, use the following (running `make` can take a while):
 git clone https://github.com/ostreedev/ostree ~/ostree
 cd ~/ostree
 git submodule update --init
+
 # for Fedora, CentOS, RHEL
 sudo yum install -y automake bison e2fsprogs-devel fuse-devel libtool xz-devel zlib-devel
```

Comment:
systemd-devel

Reply:
libbassuan-devel, gpgme-devel, libseccomp-devel

Reply:
I'm not finding libbassuan-devel....  I've added the others.

Reply:
and do we know if they have equivalents in Debian/Ubuntu?  @lsm5?

Reply:
Found 'em over there, they' lib*-dev